013-quarkus-oidc-restclient/README.md

```diff
@@ -0,0 +1,4 @@
+Module that covers the RestClient and OIDC integration using Keycloak container.
+There are two applications:
+- Ping application that will invoke the Pong application using the RestClient and will return the expected "ping pong" output.
+- Pong application that will return the "pong" output.
```

Comment:
Can you add case
A) when pong endpoint is accessed 
 - without expected Bearer token
 - with incorrect Bearer token

B) when ping endpoint is accessed without expected token (not sure if possible in this maven module or new module would be required)
?

Also please mention rest / reactive / lookup-auth variants of the tests

Reply:
I've added a couple of more scenarios to cover the incorrect bearer cases and updated the readme. 
Let me know if it looks good now.